queue_services/entity-emailer/src/entity_emailer/email_processors/amalgamation_notification.py

```diff
@@ -0,0 +1,186 @@
+            logger.error('Failed to get corrected registration statement pdf for filing: %s', filing.id)
+        else:
+            certificate_encoded = base64.b64encode(certificate.content)
+            pdfs.append(
+                {
+                    'fileName': 'Certificate Of Amalgamation.pdf',
+                    'fileBytes': certificate_encoded.decode('utf-8'),
+                    'fileUrl': '',
+                    'attachOrder': attach_order
+                }
+            )
+            attach_order += 1
+        # add notice of articles
+        noa = requests.get(
+            f'{current_app.config.get("LEGAL_API_URL")}/businesses/{business["identifier"]}/filings/{filing.id}'
+            '?type=noticeOfArticles',
+            headers=headers
+        )
+        if noa.status_code != HTTPStatus.OK:
+            logger.error('Failed to get noa pdf for filing: %s', filing.id)
+        else:
+            noa_encoded = base64.b64encode(noa.content)
+            pdfs.append(
+                {
+                    'fileName': 'Notice of Articles.pdf',
+                    'fileBytes': noa_encoded.decode('utf-8'),
+                    'fileUrl': '',
+                    'attachOrder': attach_order
+                }
+            )
+            attach_order += 1
+    return pdfs
+
+
+def process(email_info: dict, token: str) -> dict:  # pylint: disable=too-many-locals, , too-many-branches
+    """Build the email for Amalgamation notification."""
+    logger.debug('filing_notification: %s', email_info)
+
+    # get template vars from filing
+    filing, business, leg_tmz_filing_date, leg_tmz_effective_date = \
+        get_filing_info(email_info['data']['filing']['header']['filingId'])
+    filing_type = filing.filing_type
+    status = filing.status
+    filing_name = filing.filing_type[0].upper() + ' '.join(re.findall('[a-zA-Z][^A-Z]*', filing.filing_type[1:]))
+
+    template = Path(f'{current_app.config.get("TEMPLATE_PATH")}/AMALGA-{status}.html').read_text()
+    filled_template = substitute_template_parts(template)
+    # render template with vars
+    jnja_template = Template(filled_template, autoescape=True)
+    filing_data = (filing.json)['filing'][f'{filing_type}']
+    html_out = jnja_template.render(
+        business=business,
+        filing=filing_data,
+        header=(filing.json)['filing']['header'],
+        filing_date_time=leg_tmz_filing_date,
+        effective_date_time=leg_tmz_effective_date,
+        entity_dashboard_url=current_app.config.get('DASHBOARD_URL') +
+        (filing.json)['filing']['business'].get('identifier', ''),
+        email_header=filing_name.upper(),
+        filing_type=filing_type
+    )
+
+    # get attachments
+    pdfs = _get_pdfs(status, token, business, filing, leg_tmz_filing_date, leg_tmz_effective_date)
+    if not pdfs:
+        return {}
+
+    # get recipients
+    recipients = get_recipients(status, filing.filing_json, token)
```

Comment:
missing one parameter (filing_type)

Reply:
and get_recipients will return only Registered office email address when filing is in completed status

Reply:
Can you check with Mihai/Yui why there is a difference in IA recipients and Amalgamation recipients (when filing is COMPLETED)

Reply:
I will check with him

Reply:
I checked with Yui said that `Amalgamation is same with IA and the email spreadsheet has an error.`
I just updated everything